api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -35,46 +35,53 @@
  * Tests methods on the {@link ConceptValidator} class.
  */
 public class ConceptValidatorTest extends BaseContextSensitiveTest {
-	
+
 	/**
 	 * @verifies {@link ConceptValidator#validate(Object,Errors)} test = should fail if there is a
 	 *           duplicate unretired concept name in the locale
 	 */
-	@Test(expected = DuplicateConceptNameException.class)
+
+	@Test
 	@Verifies(value = "should fail if there is a duplicate unretired concept name in the locale", method = "validate(Concept)")
 	public void validate_shouldFailIfThereIsADuplicateUnretiredConceptNameInTheLocale() throws Exception {
 		Context.setLocale(new Locale("en", "GB"));
 		Concept concept = Context.getConceptService().getConcept(5497);
 		String duplicateName = concept.getFullySpecifiedName(Context.getLocale()).getName();
-		
+
 		ConceptName newName = new ConceptName(duplicateName, Context.getLocale());
 		newName.setDateCreated(Calendar.getInstance().getTime());
 		newName.setCreator(Context.getAuthenticatedUser());
 		concept.addName(newName);
 		Errors errors = new BindException(concept, "concept");
-		new ConceptValidator().validate(concept, errors);
+
+		try {
```

Comment:
The proper thing to do here is to use junit's ExpectedException object instead of a try and catch, for an example see [here](https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/api/OrderServiceTest.java#L145)